returntocorp/semgrep-grammars
# adding kotlin integration, updating readme with instructions.

Test Plan:

1. `cd src/semgrep-kotlin` 
2. Run `make && make install`
3. Check that tests are passing.
